jupyterlab/labapp.py

```diff
@@ -580,6 +580,14 @@ class LabApp(NotebookConfigShimMixin, LabServerApp):
         document will be kept in memory forever.""",
     )
 
+    collaborative_document_save_delay = Int(
+        1,
+        allow_none=True,
+        config=True,
+        help="""The second of inactivity to prevent too frequent saving (relevant only in collaborative mode). Defaults to 1s, if None then the
```

Comment:
```suggestion
        help="""The seconds of inactivity before saving the document (relevant only in collaborative mode). Defaults to 1s, if None then the
```

Reply:
Done in https://github.com/jupyterlab/jupyterlab/pull/13392